compiler/p/codegen/OMRCodeGenerator.hpp

```diff
@@ -171,6 +171,8 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
 
    bool getSupportsIbyteswap();
 
+   bool supportsAtomicAdd() {return true;}
```

Comment:
FYI, https://github.com/eclipse/omr/pull/2999 is changing this query, it may need to be updated before merging.

Reply:
Revised in commit https://github.com/eclipse/omr/pull/3018/commits/8077bb18b5170cb1e62f23fef75258c0dc006d5d